sdk/go/client_test.go

```diff
@@ -161,16 +161,25 @@ func TestList(t *testing.T) {
 		Container().
 		From("alpine:3.16.2").
 		WithEnvVariable("FOO", "BAR").
+		WithEnvVariable("BAR", "BAZ").
 		EnvVariables(ctx)
 
 	require.NoError(t, err)
 
-	envName, err := envs[0].Name(ctx)
+	envName, err := envs[1].Name(ctx)
```

Comment:
Guessing this started returning `$PATH` at `envs[0]` instead? :)

Reply:
Exactly 😇